kobo/apps/project_ownership/models/choices.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class InviteStatusChoices(models.TextChoices):
+
+    ACCEPTED = 'accepted', 'ACCEPTED'
+    CANCELLED = 'cancelled', 'CANCELLED'
+    COMPLETE = 'complete', 'COMPLETE'
+    DECLINED = 'declined', 'DECLINED'
+    EXPIRED = 'expired', 'EXPIRED'
+    FAILED = 'failed', 'FAILED'
```

Comment:
Is there room to simplify the amount of statuses? What's the justification for each one? Could we instead delete the row in some cases?

Reply:
Different flows are relying on each status. I prefer keep all of them. 
FYI: Some rows are deleting by the garbage collector.